### ISSUE TITLE
jaxb-api dependency issue is throw when working with jdk 11.

### ISSUE THREAD

**Issue by saiteja1111n** [zhurlik/swagger-gradle-example#2]
Currently the maven task created from gradle seems to throw error with ClassNotFound for depencency on jax-b. Adding the gradle dependency doesn't seem to solve it.

**Comment by zhurlik** [comment]
Hi,

Could you provide your `jvm` and `gradle` versions, something like this:
```
./gradlew --version

------------------------------------------------------------
Gradle 7.1.1
------------------------------------------------------------

Build time:   2021-07-02 12:16:43 UTC
Revision:     774525a055494e0ece39f522ac7ad17498ce032c

Kotlin:       1.4.31
Groovy:       3.0.7
Ant:          Apache Ant(TM) version 1.10.9 compiled on September 27 2020
JVM:          11.0.11 (Ubuntu 11.0.11+9-Ubuntu-0ubuntu2)
OS:           Linux 5.11.0-25-generic amd64
```
To be able to verify it with yours versions

Thanks,
Vlad

**Comment by saiteja1111n** [comment]
Gradle version used in my project

------------------------------------------------------------
Gradle 4.10.2
------------------------------------------------------------

Build time:   2018-09-19 18:10:15 UTC
Revision:     b4d8d5d170bb4ba516e88d7fe5647e2323d791dd

Kotlin DSL:   1.0-rc-6
Kotlin:       1.2.61
Groovy:       2.4.15
Ant:          Apache Ant(TM) version 1.9.11 compiled on March 23 2018
JVM:          11 (Oracle Corporation 11+28)
OS:           Windows 10 10.0 amd64

**Comment by zhurlik** [comment]
Hi,

For me it looks like you ran into the issue that's described here [java-11-package-javax-xml-bind-does-not-exist](https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist)

You can have a look at small fix in [2f4030f918d3533b0ad3039854c0f364df3213e5](https://github.com/zhurlik/swagger-gradle-example/commit/2f4030f918d3533b0ad3039854c0f364df3213e5)

Let me know if it works for you

Thanks,
Vlad
